metadata-ingestion/src/datahub/ingestion/source/metadata/business_glossary.py

```diff
@@ -66,6 +72,8 @@ class DefaultConfig(ConfigModel):
 
 class BusinessGlossarySourceConfig(ConfigModel):
     file: str = Field(description="Path to business glossary file to ingest.")
+    enable_datahub_guid: bool = Field(
```

Comment:
Suggest renaming this field to `enable_auto_id`  to line up with the suggested `id` field for terms and term groups.

Reply:
Done